### PR TITLE
docs: use new blockquotes, fix Generics and use normal text for description

### PR DESCRIPTION
**Pull request by fraxken**

### CODE REVIEW

File: docs/Installation.md [hunk 1]
```diff
@@ -16,7 +16,7 @@
 ### Launch the server
 
 First at all, you must open a terminal and launch the server
```
Comment (Rossb0b) [inline docs/Installation.md:18]: First of all

Comment (fraxken) [reply docs/Installation.md:18]: Fait une suggestion :)

Comment (Rossb0b) [reply docs/Installation.md:18]: Juste "First of all" plutôt que "First at all" :p

Comment (fraxken) [inline docs/Installation.md:18]:
```suggestion
First of all, you must open a terminal and launch the server
```

**Review by Rossb0b — APPROVED** [review]

**Pull request merged**